### PR TITLE
Splice 707 splice 708

### PR DESCRIPTION
**Pull request by jleach4**
Applying two derby patches to support order by error checking and simple execution of values clauses...

### CODE REVIEW

File: splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/SortOperationIT.java [hunk 1]
```diff
@@ -263,6 +264,27 @@ public void testSortOperationByKey2Descending() throws Exception {
...
 	}
 
+	@Test(expected = SQLSyntaxErrorException.class)
+	public void testDerby6027() throws Exception {
+		methodWatcher.executeQuery("values 1,2 order by int(1)");
+	}
+
+	@Test(expected = SQLSyntaxErrorException.class)
+	public void orderByWithComputation() throws Exception {
+		methodWatcher.executeQuery("values 1 order by 1+0");
```
Comment (jeffcunningham) [inline splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/SortOperationIT.java]: You could catch the exception here and assert its sql code is the new one you added.

**Comment by jeffcunningham** [comment]
Other than the commented test improvement, +1

**Comment by scottfines** [comment]
Please create a separate pull request for each distinct ticket. That way we can keep our patch-sets clean, and other people wanting to merge patches into their fork will know what they are getting.

**Comment by jyuanca** [comment]
+1

**Pull request merged**